crates/test-macro/src/lib.rs

```diff
@@ -80,6 +80,12 @@ pub fn wasm_bindgen_test(
         None => quote! { ::core::option::Option::None },
     };
 
+    let ignore_str = match ignore.clone() {
+        Some(Some(_)) => "$",
+        Some(None) => "$",
```

Comment:
```suggestion
        Some(_) => "$",
```

Reply:
Yes, this was a left over, that I didn't clean up, sorry.